index.js

```diff
@@ -1,6 +1,8 @@
 //https://github.com/angular/angular.js/pull/10732
 
-var angular = require('angular');
+if (typeof angular !== 'undefined') {
```

Comment:
Where do you see to assigning it to `undefined`? It's only a conditional.
It's widely popular in npm comunity for AMD/UMD/CommonsJs/ES6module support.